vision/interface.py

```diff
@@ -23,3 +23,18 @@ def __iter__(self):
     #updates rects list
     def update(self, bounding_boxes):
         self.bounding_boxes = bounding_boxes
+
+    #This is the functiont that the teams should call
+    #safe_update handles all multiprocessing options
+    #CAUTION: may not run only work on Linux/MacOS  
```

Comment:
Switch to docstring, move under definition and describe the bounding box parameter for consistency and to make sure code editors pick up on it.